gather

```diff
@@ -49,41 +39,86 @@ def run(options=None):
     parents = get_parent_domains(options)
 
 
+    # Import the gatherer(s).
+    sources = options["_"][0].split(",")
+
     # Open CSV writer.
-    gathered_filename = "%s/%s.csv" % (utils.results_dir(), source)
-    gathered_file = open(gathered_filename, 'w', newline='')
-    gathered_writer = csv.writer(gathered_file)
-    gathered_writer.writerow(["Domain", "Base Domain"])
+    if len(sources) > 1:
+        filename = "gathered"
+    else:
+        filename = sources[0] # backwards compatibility, mostly
```

Comment:
For backwards compatibility, if only one source is used then the filename will be named after that source (e.g. `censys.csv`), and if multiple are used then the filename will be `gathered.csv`.